AnkiDroid/src/main/java/com/ichi2/compat/CompatV30.kt

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2022 Nishant Bhandari <nishantbhandari0019@gmail.com>
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software
+ * Foundation; either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+ * PARTICULAR PURPOSE. See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.ichi2.compat
+
+import android.annotation.TargetApi
+import android.view.Window
+import android.view.WindowInsets
+import android.view.WindowInsetsController
+
+/** Implementation of [Compat] for SDK level 30 and higher. Check [Compat]'s for more detail.  */
+@TargetApi(30)
+class CompatV30 : CompatV26(), Compat {
```

Comment:
```suggestion
class CompatV30 : CompatV29(), Compat {
```

must extend the next lower-one for the overriding to chain correctly and include the 29 overrides (and 28, and 27) vs 26

Reply:
Done.